templates/search/blocks/search_results.html

```diff
@@ -24,6 +24,7 @@
         {% include './search_results__selected-filters.html' %}
 
         {% if form.errors or not paginator.count %}
+            {# No results were found due to criteria not staisfying the refine results filters #}
```

Comment:
I think we should remove this, as it might just be that there were no results.

Reply:
Removed.